lib/trln_argon/solr_document/syndetics_data.rb

```diff
@@ -72,9 +72,9 @@ def build_syndetics_query(params)
 
       def get_params(client = 'trlnet', format_spec = 'XML.XML')
         params = {
-          isbn: [fetch(TrlnArgon::Fields::ISBN_NUMBER, []).first].compact,
-          oclc: fetch(TrlnArgon::Fields::OCLC_NUMBER, []).dup,
-          upc: [upc.first].compact
+          isbn: [primary_isbn.first].compact,
+          oclc: primary_oclc.dup,
```

Comment:
@kazymovae I believe this should be `oclc: [primary_oclc.first].compact,` just like `primary_isbn` and `primary_upc`.